README.md

```diff
@@ -7,3 +7,8 @@ A simple web crawler. This leverages Go's lightweight goroutines to achieve high
 and enjoy.
 
 OSX binary is available in the release.
```

Comment:
Any further info on How user can use this binary?
Any steps needed?

Reply:
Altho, downloading and running a binary should be obvious to any dev but still added that you can use the `./crawlerOSX` syntax for assistance.